src/components/PlayListVideoPage.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from "react";
+import {useParams} from "react-router";
+import {Link} from "react-router-dom";
+import {useVideo} from "../context/video-context";
+import {VideoCard} from "./VideoCard";
+import {VideoList} from "./VideoList";
+import {VideoPage} from "./VideoPage";
+import "./PlayListVideoPage.css";
+export function PlayListVideoPage() {
+  let {playlistid} = useParams();
+  const {
+    value: {data, playLists},
+  } = useVideo();
+
+  console.log(playLists);
+  console.log(playlistid);
+  console.log(playLists.filter((prev) => prev.id == playlistid));
+  const getPlayListFirstVideo = (playlistid) => {
```

Comment:
These small helper functions can go in `src/utils/playlist.js` and export the functions in the `src/utils/index.js`

```suggestion
// file => src/utils/index.js

export { getAllVideos, getPlaylistFirstVideo } from './playlist.js';
```